state/s3.yaml

```diff
@@ -213,12 +286,34 @@ Resources:
           Condition:
             StringEquals:
               's3:x-amz-acl': 'bucket-owner-full-control'
+  BucketPolicyCloudTrailCrossAccountWrite:
```

Comment:
This allows any AWS account to publish their logs to your AWS account, correct? Is there no way to restrict based on OU/Account Id/...?

Reply:
removed